code/modules/cargo/bounties/assistant.dm

```diff
@@ -201,10 +201,10 @@
 	wanted_types = list(/obj/item/restraints/legcuffs/bola)
 
 /datum/bounty/item/assistant/metalshields
-	name = "Metal Shields"
+	name = "Metal Shields" //I didnt realise how much work it was to make these, you need 6 Cloth, 9 Leather, Tools, 30 Metal, and 3 Cable Coil Stacks for this.
 	description = "NT is testing the effects of electricity on clowns wielding metal shields. We have clowns, and we have electricity. Send us the shields."
-	reward = 1400
-	required_count = 4
+	reward = 2300
```

Comment:
raise this up to 5k for 2 imo

Reply:
I raised it for 2 for 3k. Cargo can print the tools, metal, and cables for this. You can easily get cable from bedsheets, but the leather is the thing that slows it down, which you can get from both ghetto botany and normal botany with enough effort.